decentraland/unity-explorer
# feat: Add Sidebar Missing Animations + fixes

## What does this PR change?

We are adding the proper animations for the notifications and the emotes button as well as a couple of minor visual fixes to them + a fix for the button selection on the notifications button.

## How to test the changes?

Just hover the notification button and the emotes button. The new animations should show.
Also the notification button should open correctly the notification menu and when closing it animate back to its unselected state.
